.pre-commit-config.yaml

```diff
@@ -17,7 +17,7 @@ repos:
     -   id: end-of-file-fixer
 
 -   repo: https://github.com/ambv/black
-    rev: 22.1.0
+    rev: 22.3.0
```

Comment:
Could you pin the version in `.github/workflows/build.yml` as well? 

Does this require any updates to other files?

Reply:
Sure, I've pinned the version of `black` in github `build` workflow and force-pushed the new commit. Please let me know if more changes are needed, I can't see the results of the internal Phabricator build.

Reply:
Thanks for the changes!